lib/fog/shared_file_system/openstack/requests/list_availability_zones.rb

```diff
@@ -0,0 +1,24 @@
+module Fog
+  module SharedFileSystem
+    class OpenStack
+      class Real
+        def list_availability_zones()
+          request(
+            :expects => 200,
+            :method  => 'GET',
+            :path    => 'availability-zones'
```

Comment:
I suggest to build support for microversions prior 2.6, too with `os-availability-zone`, check with `microversion_newer_than?('2.6')`